src/fhirdefs/load.ts

```diff
@@ -25,13 +25,20 @@ export async function loadDependency(
   FHIRDefs: FHIRDefinitions,
   cachePath: string = path.join(os.homedir(), '.fhir', 'packages')
 ): Promise<FHIRDefinitions> {
+  if (/(?=.*[A-Z])/.test(packageName)) {
+    logger.warn(
+      `${packageName} contains upper-case characters, which is discouraged. SUSHI will download ${packageName.toLowerCase()}#${version} if the package is not found in local cache.`
```

Comment:
I think `uppercase` (all one word) is more common. Also I think this error could in theory be slightly inaccurate, since if they are looking for a `dev` package for example, we aren't going to download anything. I think it could be more accurate to just say "SUSHI will use `${packageName.toLowerCase()}` as the packages name."

Reply:
I hadn't thought of that! I changed the warning message in commit `855cd26`.